spanner/examples_test.go

```diff
@@ -767,3 +772,19 @@ func ExampleNewReadWriteStmtBasedTransaction() {
 		time.Sleep(delay)
 	}
 }
+
+func ExampleClient_usingEmulator() {
+	ctx := context.Background()
+	emulatorAddr := os.Getenv("SPANNER_EMULATOR_HOST")
```

Comment:
We already have this code inside the client and the users only need to set SPANNER_EMULATOR_HOST: https://github.com/googleapis/google-cloud-go/blob/master/spanner/client.go#L148-L157

Please take a look at this PR as a reference: https://github.com/googleapis/google-cloud-go/pull/4656/files